controller/access/access.go

```diff
@@ -759,7 +759,7 @@ func CompileUriPermitsMapping() {
 				"v1/scan/platform/platform",
 				"v1/scan/asset",
 				"v1/vulasset",
-				// "scanasset5", // TODO: local dev only
+				"v1/scan/asset/images",
```

Comment:
Please update access/access_test.go & Permission_API.txt for this new API (see "/v1/vulasset" for reference)

Reply:
added in access/access_test.go.
added in Permission_API.txt